crates/bevy_reflect/derive/src/container_attributes.rs

```diff
@@ -509,6 +533,24 @@ impl ContainerAttributes {
         }
     }
 
+    pub fn get_clone_impl(&self, bevy_reflect_path: &Path) -> Option<proc_macro2::TokenStream> {
```

Comment:
Where is this used? I couldn't understand this part.
I get the `clone_impl` on struct,tuple,enum,value,etc. but not this.

Or is this the top-level impl?

Reply:
It's used by `derive_data.rs`. I could probably move this logic into that file directly, since it's the only place where it's used.